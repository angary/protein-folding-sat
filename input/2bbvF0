101100100110001
