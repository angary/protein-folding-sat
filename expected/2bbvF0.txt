Maximum contacts found for 101100100110001 using Lingeling: 4
Lingeling time taken: 1151.3368799686432
Lingeling runs required: 6
Maximum contacts found for 101100100110001 using gurobi: 4
Gurobi time taken: 1520.1160073280334
Maximum contacts found for 101100100110001 using Lingeling: 4
Lingeling time taken: 1146.4272141456604
Lingeling runs required: 6
Maximum contacts found for 101100100110001 using gurobi: 4
Gurobi time taken: 1512.1539590358734
